beekeeper-metadata-cleanup/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+  xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>beekeeper-parent</artifactId>
+    <groupId>com.expediagroup</groupId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>beekeeper-metadata-cleanup</artifactId>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.expediagroup</groupId>
+      <artifactId>beekeeper-core</artifactId>
+      <version>${project.version}</version>
+      <exclusions>
+        <!-- if this is on classpath then not-null checks in objects persisted via JPA are ignored -->
```

Comment:
eh?

Reply:
this is copied from somewhere or not?

Reply:
yeah, copied from the pom for beekeeper-cleanup

Reply:
ok so you want to keep just in case you will add non-null checks?

Reply:
yeah

Reply:
Oh, the pain that was involved in figuring this one out, it's all coming back to me now, make it stop!